packages/perseus/src/components/math-input.tsx

```diff
@@ -375,7 +386,33 @@ class MathInput extends React.Component<Props, State> {
     }
 }
 
-MathInput.contextType = MathInputI18nContext;
+// We need to have two contexts (one for Perseus and one for MathInput), so we
+// add a wrapper around the MathInput component to provide the MathInput context
+// to it.
+class MathInput extends React.Component<Props, State> {
+    static contextType = MathInputI18nContext;
+    declare context: React.ContextType<typeof MathInputI18nContext>;
+
+    focus() {
+        // @ts-expect-error - types aren't valid here, but this API is exposed.
+        this.refs.input.focus();
+    }
+
+    insert(value: any) {
+        // @ts-expect-error - types aren't valid here, but this API is exposed.
+        this.refs.input.insert(value);
+    }
+
+    render() {
+        return (
+            <InnerMathInput
+                {...this.props}
+                ref="input"
```

Comment:
Can we just use a regular `React.createContext<>()` here instead of a string ref?